src/options/block-indent.js

```diff
@@ -72,7 +72,7 @@ let option = {
         var lastIndex = spaces.lastIndexOf('\n');
 
         // Do not continue if there is no line break:
-        if (lastIndex < 0) return;
+        if (lastIndex === -1) return;
```

Comment:
@jdalton, out of curiosity, there'll be no bug/issue with this line, it's just a nitpick, right?

Reply:
Ah, sorry about that. No bug. I usually try to avoid unfocused changes like that. 

Let me know if you spot others like it and I’ll revert them.